mlir/lib/Conversion/SCFToOpenMP/SCFToOpenMP.cpp

```diff
@@ -390,6 +392,12 @@ struct ParallelOpLowering : public OpRewritePattern<scf::ParallelOp> {
 
     // Create the parallel wrapper.
     auto ompParallel = rewriter.create<omp::ParallelOp>(loc);
+    if (numThreads > 1) {
+      rewriter.setInsertionPoint(ompParallel);
+      mlir::Value numThreadsVar = rewriter.create<LLVM::ConstantOp>(
+          loc, rewriter.getI32IntegerAttr(numThreads));
```

Comment:
Nit: I'd rather use 64-bit integers unless there is a strong reason not to.

Reply:
The entrypoint `__kmpc_push_num_threads` takes a `int32_t` for the number of threads coming from the `num_threads` clause.  So, a 32bit int might be fine.

Reply:
I agree, I think it should be 32 bit.